### PR TITLE
#55: Select list for template downloader

### PR DESCRIPTION
**Pull request by Benedito0**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-pulse/issues/55|
| Helpers | zeplin: https://app.zeplin.io/project/5db732f9c02f1e8327288741/screen/5dfa634d9ca8da976f6bb469 |
| Animated GIF |AWad6BhQLf|

### CODE REVIEW

File: vue/components/ui/molecules/index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { Tabs } from "./tabs/tabs.vue";
 import { Search } from "./search/search.vue";
 import { Select } from "./select/select.vue";
 import { UploadArea } from "./upload-area/upload-area.vue";
+import { SelectList } from "./select-list/select-list.vue";
```
Comment (Benedito0) [inline vue/components/ui/molecules/index.js]: Order

File: vue/components/ui/molecules/index.js [hunk 2]
```diff
@@ -32,6 +33,7 @@ const install = Vue => {
     Vue.component("table-ripe", Table);
     Vue.component("tabs", Tabs);
     Vue.component("upload-area", UploadArea);
+    Vue.component("select-list", SelectList);
```
Comment (Benedito0) [inline vue/components/ui/molecules/index.js]: Order

File: vue/components/ui/molecules/index.js [hunk 3]
```diff
@@ -50,7 +52,8 @@ export {
     Select,
     Table,
     Tabs,
-    UploadArea
+    UploadArea,
+    SelectList
```
Comment (Benedito0) [inline vue/components/ui/molecules/index.js]: Order

File: vue/components/ui/molecules/select-list/select-list.vue [hunk 4]
```diff
@@ -3,16 +3,25 @@
...
         </p>
+        <div class="select-list-seach">
+            <search
+                class="search-containter"
+                v-bind:icon-visible="true"
+                v-bind:clear-visible="true"
+                v-bind:value.sync="searchKeyData"
+            />
+        </div>
+
```
Comment (NFSS10) [inline vue/components/ui/molecules/select-list/select-list.vue]: Remove space

File: vue/components/ui/molecules/select-list/select-list.vue [hunk 5]
```diff
@@ -36,6 +45,14 @@
     padding: 5px 0px 0px 0px;
 }
 
+.select-list > .select-list-seach {
+    background-color: #ffffff;
+    border: 1px solid #e4e8f0;
```
Comment (NFSS10) [inline vue/components/ui/molecules/select-list/select-list.vue]: Use color variables when possible:
```
background-color: $white;
border: 1px solid $light-white;
```

**Pull request merged**